README.md

```
@@ -182,21 +154,18 @@ TODO: Document main user flows:
 
 ### Prerequisites
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)